### PR TITLE
[Dialogs] MDCAlertController should announce 'alert' in VO

### LINKED ISSUE #821: MDCAlertController should announce 'alert' in VO
Internal Link:
b/32564172#comment3

### PR DESCRIPTION
**Pull request by sread94**
This closes #821.

### CODE REVIEW

**Comment by ianegordon** [comment]
Hi @sread94, thanks for the PR. Could you sign the CLA and post to this thread so we can take a look?

File: components/Dialogs/src/private/MaterialDialogsStrings.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```
Comment (ianegordon) [inline components/Dialogs/src/private/MaterialDialogsStrings.h]: s/2016/2017/

Time marches on.

File: components/Dialogs/src/private/MaterialDialogsStrings_table.h [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```
Comment (ianegordon) [inline components/Dialogs/src/private/MaterialDialogsStrings_table.h]: s/2016/2017/

Time marches on.

**Review by ianegordon — CHANGES_REQUESTED** [review]
Hi Sarah.  This looks good.  If you could fix the two typos, we'll get this merged.  Many Thanks!

File: components/Dialogs/src/private/MaterialDialogsStrings.h (hunk 1 above)
Comment (sread94) [reply components/Dialogs/src/private/MaterialDialogsStrings.h]: Done.

File: components/Dialogs/src/private/MaterialDialogsStrings_table.h (hunk 2 above)
Comment (sread94) [reply components/Dialogs/src/private/MaterialDialogsStrings_table.h]: Done.

**Comment by ianegordon** [comment]
I'm not able to get the announcement in VO mode on an iPhone 5 @ iOS 8.3.
Do you have a moment to catch up offline for a demo?

**Comment by ianegordon** [comment]
This PR works as expected on an iPhone 7 @ iOS 10.x.

We will open an issue to investigate further, but will land this PR now.

Thank you @sread94 !

**Review by ianegordon — APPROVED** [review]

**Pull request merged**